src/tlo/methods/hsi_event.py

```diff
@@ -172,8 +172,12 @@ def never_ran(self) -> None:
         logger.debug(key="message", data=f"{self.__class__.__name__}: was never run.")
 
     def post_apply_hook(self) -> None:
+        """Do any required processing after apply() completes."""
+        pass
```

Comment:
```suggestion
```
[The `pass` is not strictly required here as the docstring suffices to make this syntactically correct as a function definition by having an indented block](https://docs.astral.sh/ruff/rules/unnecessary-placeholder/).